src/DIRAC/Core/Utilities/JDL.py

```diff
@@ -157,3 +161,156 @@ def dumpCFGAsJDL(cfg, level=1, tab="  "):
                 contents.append("%s};" % indent)
     contents.append("%s]" % (tab * (level - 1)))
     return "\n".join(contents)
+
+
+def loadJDLasJob(jdl: str):
+
+    jdl = jdl.strip()
+    if not jdl.startswith("[") and not jdl.endswith("]"):
+        jdl = f"[{jdl}]"
+
+    jobDescription = transformParametricJobIntoParsableOne(ClassAd(jdl))
+
+    if jobDescription.lookupAttribute("StdOutput"):
+        stdout = jobDescription.getAttributeString("StdOutput")
+    else:
+        stdout = "std.out"
+
+    if jobDescription.lookupAttribute("StdError"):
+        stderr = jobDescription.getAttributeString("StdError")
+    else:
+        stderr = "std.err"
+
+    job = Job(stdout=stdout, stderr=stderr)
+
+    errors = []
+
+    # Mandatory fields in the job description
+    if jobDescription.lookupAttribute("Executable"):
+        executable = jobDescription.getAttributeString("Executable")
+
+        if jobDescription.lookupAttribute("Arguments"):
+            res = job.setExecutable(executable, jobDescription.getAttributeString("Arguments"))
+        else:
+            res = job.setExecutable(executable)
+
+        if not res["OK"]:
+            errors.append(res["Message"])
+
+    if jobDescription.lookupAttribute("ExecutionEnvironment"):
+        res = jobDescription.getListFromExpression("ExecutionEnvironment")
+        environmentDict = {}
+        for element in res:
+            key, value = element.replace(" ", "").split("=")
+            environmentDict[key] = value
+        job.setExecutionEnv(environmentDict)
+        jobDescription.deleteAttribute("ExecutionEnvironment")
+
+    if jobDescription.lookupAttribute("JobType"):
+        # TODO: should we set the job type, put a warning and keep going or return S_ERROR ?
```

Comment:
To discuss with @fstagni 

Last time we talked about this we said that you could:
- not look these attributes up
- at the end of the function, check what remains in `jobDescription`, so basically parameters such as `JobType` or `Owner`.
- for each remaining parameter that was processed, display a warning: `Parameter x has been found but has been ignored` or something.

Reply:
Warnings I am afraid are not very effective.